src/main/java/com/sngular/kloadgen/sampler/schemaregistry/schema/ApicurioParsedSchema.java

```diff
@@ -1,17 +1,31 @@
 package com.sngular.kloadgen.sampler.schemaregistry.schema;
 
+import com.sngular.kloadgen.sampler.schemaregistry.adapter.impl.ApicurioParsedSchemaMetadata;
+import com.sngular.kloadgen.sampler.schemaregistry.adapter.impl.ParsedSchemaAdapter;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
 @NoArgsConstructor
-public class ApicurioParsedSchema {
+public class ApicurioParsedSchema extends ParsedSchemaAdapter {
```

Comment:
I think that this class is not necessary. It is only used as argument to instantiate ApicurioParsedSchemaMetadata object, but that argumento must be replace by io.apicurio.registry.resolver.ParsedSchema object type

Reply:
class deleted